lib/utils/transcription-utils.js

```diff
@@ -590,6 +590,8 @@ module.exports = (logger) => {
           {AZURE_SPEECH_SEGMENTATION_SILENCE_TIMEOUT_MS: azureOptions.speechSegmentationSilenceTimeoutMs}),
         ...(azureOptions.languageIdMode &&
           {AZURE_LANGUAGE_ID_MODE: azureOptions.languageIdMode}),
+        ...(azureOptions.postProcessing &&
+          {AZURE_POST_PROCESSING_OPTION: azureOptions.postProcessing}),
```

Comment:
We dont have this options defined in verb-specs yet I dont think.  This PR should come after the PR to add it there